sma-android/.gitignore

```diff
@@ -1,2 +1,3 @@
 src/main/resources/lib
 src/main/libs/com.fiskaly.kassensichv.sma.jar
+out/production/resources/lib
```

Comment:
Is this really needed?

Reply:
Was not really needed, I had some issues with this folder coming up, but I undid the changes as it didn't happen again yet.